test/dapis/DapiServer.sol.js

```diff
@@ -302,6 +302,21 @@ describe('DapiServer', function () {
     return [data, signature];
   }
 
+  async function encodeAndDomainSignData(decodedData, templateId, timestamp) {
```

Comment:
Maybe the function above (`encodeAndSignData` on line 293) should also use `templateId` instead of `requestHash` as name for the parameter.

Reply:
Those functions are a mess, I'll revisit them in https://github.com/api3dao/airnode-protocol-v1/issues/39